site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -458,94 +547,10 @@ function printPreview(target){
-							    <fieldset><legend>$tlang.getString('gen.previous') $pagesize</legend>
-								<input type="submit" name="eventSubmit_doList_prev" value="&lt;" disabled="disabled" />
-                                </fieldset>
-								<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
-							</form>
-						#end
-					#end
-					<form name="pagesizeForm" class="inlineForm" method="post" action="#toolForm("$action")">
-						<input type="hidden" name="eventSubmit_doChange_pagesize" value="changepagesize" />
-						<label for="selectPageSize" class="skip">$tlang.getString("sitegen.siteinfolist.view.pagesize")</label>
-						<select id="selectPageSize" name="selectPageSize" onchange="document.pagesizeForm.submit();">
-							#foreach ($i in $!pagesizes)
-								<option value="$i" #if($pagesize == $i) selected="selected" #end>$tlang.getString("sitegen.siteinfolist.show") $i $tlang.getString("sitegen.siteinfolist.itemspage")</option>
-							#end
-						</select>
-						<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
-					</form>
-					#if ($pagesize != 0)
-						#if ($goNPButton == "true")
-							<form name="nextpageForm" class="inlineForm" method="post" action="#toolForm("$action")">
-							    <fieldset><legend>$tlang.getString('gen.next') $pagesize</legend>
-								<input type="submit" name="eventSubmit_doList_next" value="&gt;" title="$tlang.getString('gen.next') $pagesize" />
-                                </fieldset>
-								<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
-							</form>
-						#else
-							<form name="nextpageForm" class="inlineForm" method="post" action="#toolForm("$action")">
-							    <fieldset><legend>$tlang.getString('gen.next') $pagesize</legend>
-								<input type="submit" name="eventSubmit_doList_next" value="&gt;" disabled="disabled" />
-                                </fieldset>
-								<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
-							</form>
-						#end
-						#if ($goLPButton == "true")
-							<form name="lastpageForm" class="inlineForm" method="post" action="#toolForm("$action")">
-							    <fieldset><legend>$tlang.getString('gen.last')</legend>
-								<input type="submit" name="eventSubmit_doList_last" value="&gt;|" title="$tlang.getString('gen.last')" />
-                                </fieldset>
-								<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
-							</form>
-						#else
-							<form name="lastpageForm" class="inlineForm" method="post" action="#toolForm("$action")">
-							    <fieldset><legend>$tlang.getString('gen.last')</legend>
-								<input type="submit" name="eventSubmit_doList_last" value="&gt;|" disabled="disabled" />
-                                </fieldset>
-								<input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
-							</form>
-						#end
-					#end
-				</div>
-			</div>
+    #if ($!viewRoster)
```

Comment:
The indentation here looks a little odd. If it's not right fix it but don't worry if it's not obvious or would need to to re-indent lots of lines.